feature/lldp/otg_tests/core_lldp_tlv_population_test/metadata.textproto

```diff
@@ -4,3 +4,4 @@
 uuid: "1be40a09-1f0e-4891-aa7a-12ea8e8ee02f"
 plan_id: "RT-6.1"
 description: "Core LLDP TLV Population"
+testbed: TESTBED_DUT_ATE_2LINKS
```

Comment:
This is a DUTDUT test and should match feature/lldp/ate_tests/core_lldp_tlv_population_test/metadata.textproto